portal/models/message.py

```diff
@@ -79,6 +79,18 @@ def log_message(message, app):
 EMAIL_FOOTER = "</body></html>"
 
 
+def extra_headers():
+    """Function to deliver any configured extra_headers for outbound email
+
+    Returns an empty dict, or one including any desired extra_headers, such
+    as a ``List-Unsubscribe`` header.
+
+    """
+    # TN-2386 Include List-Unsubscribe header to improve spam scores
+    return {'List-Unsubscribe': "<mailto:{}?subject=unsubscribe>".format(
+        current_app.config['CONTACT_SENDTO_EMAIL'])}
```

Comment:
NB `CONTACT_SENDTO_EMAIL` defaults to `None`

May want to omit inclusion of this header if unconfigured, otherwise every system (including dev) will require this config value (or trigger an exception on send).